src/main/charts/confluence/templates/_helpers.tpl

```diff
@@ -178,6 +178,15 @@ The command that should be run by the nfs-fixer init container to correct the pe
 {{- end }}
 {{- end }}
 
+{{/*
+The command that will be run to properly shutdown Confluence tomcat
+*/}}
+{{- define "confluence.shutdown.command" -}}
+{{- if .Values.confluence.shutdown.command }}
+{{ .Values.confluence.shutdown.command }}
+{{- else }}export CONNIE_PID=$(pgrep -f org.apache.catalina.startup.Bootstrap | sort | head -n 1); echo $CONNIE_PID > /opt/atlassian/confluence/work/catalina.pid && /bin/su confluence -c /opt/atlassian/confluence/bin/stop-confluence.sh; tail --pid=$CONNIE_PID -f /dev/null{{- end }}
```

Comment:
This is the command to shutdown Confluence properly

1. Get the PID of Confluence (the PID of the only running tomcat process)
2. Write it to the file and launch the tomcat stop script
3. Wait until the process has been killed

Reply:
This can be simplified, there is a script /opt/atlassian/support/common.sh that will do the pgrep part and populate $APP__PID.

That said, I'm going to modify the entrypoint util to dump a pid in the app home, so we can use that from now on. And I'm also going to add a dedicated shutdown util to each image.

I'd say merge this for now, and I'll port this across to the Docker image when I do that.